_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+  annotate("segment", x=2021.45, xend= 2021.6, y=31, yend=40, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=40, yend=40, color="grey") +
+  #LinkedIn
+  annotate("segment", x=2021.15, xend= 2021.45, y=28, yend=28, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=28, yend=34, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=34, yend=34, color="grey") +
+  #Snapchat
+  annotate("segment", x=2021.15, xend= 2021.45, y=25, yend=25, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=25, yend=29, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=29, yend=29, color="grey") +
+  #Twitter
+  annotate("segment", x=2021.15, xend= 2021.9, y=24, yend=24, color="grey") +
+  #Whatsapp
+  annotate("segment", x=2021.15, xend= 2021.45, y=23, yend=23, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=23, yend=18, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=18, yend=18, color="grey") +
+  #TikTok
+  annotate("segment", x=2021.15, xend= 2021.45, y=21, yend=21, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=21, yend=15, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=15, yend=15, color="grey") +
+  #Reddit
+  annotate("segment", x=2021.15, xend= 2021.45, y=18, yend=18, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=18, yend=10, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=10, yend=10, color="grey") +
+  #Nextdoor
+  annotate("segment", x=2021.15, xend= 2021.45, y=13, yend=13, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=13, yend=6, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=6, yend=6, color="grey")
+
+p
+
+```
+
+## Enhancements and alternative visualizations
+
+### Possible changes in the plot
+
+When looking at the graph by Auxier and Anderson, the first thing that can come to our minds is that, although the evolution through the last 9 year appears to be clear, the discriminability could be better. There are categories with the same color as others, and the distribution seems to be only 'reasonable' for the top two Social Networks, YouTube and Facebook, as they are depicted with their corporative colors. This is specially problematic if we take a look at the mid-bottom part of the chart, which compiles a lot of values very close to each other, resulting into some potential confusion at identifying each line or point with their respective category. 
+
+Another way of improving each category's identification with the lines could be expanding the plot margins a little bit to the bottom, to avoid the excessive proximity between points that also results into very close reference lines, but we didn't attempt to do it because excessive height for temporal series plots may distort the evolution shown, making it too difficult to visually see. 
+
+```{r, include=FALSE}
```

Comment:
Is this chunk needed? It repeats the same code above, right? If so, please remove it.

Reply:
And if not, if this takes the previous plot at a specific stage, I would be better to save the plot at that stage in a different variable (say, `p_base`), and then reuse it here. No need for this chunk.